Setup/UpgradeSchema.php

```diff
@@ -134,6 +134,18 @@ public function upgrade(
                 ]
             );
         }
+        if (version_compare($context->getVersion(), '3.1.1', '<=')) {
+            $setup->getConnection()->addColumn(
+                $setup->getTable('quote'),
+                'delivery_options',
+                [
+                    'type' => Table::TYPE_TEXT,
+                    'nullable' => true,
+                    'comment' => 'MyParcel delivery options',
+                ]
+            );
+        }
+
```

Comment:
Volgens mij is de aanpassing in de UpgradeSchema niet meer nodig?

Reply:
Klopt wist niet meer dat die erbij zat. Done now